### DISCUSSION TITLE
Selecting wav files uploaded to LittleFS

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_midi_sampler#65] [Q&A]
Hi,
Is there any way to select between multiple wav files uploaded to LittleFS using the midi controller itself?
How many number of samples(If they are 1 or 2 seconds long) might be possible to load to the LittleFS, maximum?

**Comment by marcel-licence** [comment]
Hi,

there are different partition schemes available:
![grafik](https://user-images.githubusercontent.com/65245426/183478899-d8febf4a-ba96-48a1-996f-de8526cb065d.png)

Waves are stored in 16bit@44100Hz -> 705,6kBit/s
That would result in approx 1.48s per 1MBit of Memory.
